### DISCUSSION TITLE
How are paths in index.html managed?

### DISCUSSION THREAD

**Discussion by eyyyyyyy3** [tauri-apps/tauri#8256] [Q&A] [answered]
I've noticed that index.html has this line 
`   <link rel="icon" type="image/png" href="/icon.png" />`

and that the href points to /icon.png but shouldn't it acutally be /src-tauri/icons/icon.png ?
Or is it public/icon.png and if yes why is it enough to just specify icon.png

**Comment by FabianLars** [comment, marked as answer]
> and that the href points to /icon.png but shouldn't it acutally be /src-tauri/icons/icon.png ?

Nope, the src-tauri folder contents are not part of your frontend.

>  Or is it public/icon.png and if yes why is it enough to just specify icon.png

Yep. That's just how vitejs (or most bundlers) work. Files from the public dir will be copied into the root of the distributed files.
